### PR TITLE
Reset instructions

### PR DESCRIPTION
**Pull request by alexhq**

### CODE REVIEW

File: docs/connecting.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Connecting to Chronicle
+
+## How to connect to Chronicle running in Kubernetes
+
+When installed with the default values, Chronicle will be accessible via an
+internal cluster service. This means that Chronicle will only be accessible
+from within the Kubernetes cluster.
+
+This service is named `<deployment-name>-chronicle-on-sawtooth-chronicle-api`
+and exposes port `9982
```
Comment (mtbc) [inline docs/connecting.md:10]:
```suggestion
and exposes port `9982`
```

**Review by suchapalaver — APPROVED** [review]

File: docs/restarting.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# Resetting Chronicle
```
Comment (mtbc) [inline docs/restarting.md:1]: Is this file intended?

**Comment by mtbc** [comment]
Should these be referenced in some other page, maybe listed among "Operating" in `mkdocs.yml` or somesuch?

File: docs/resetting.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    ```
+
+Next you need to delete the Sawtooth data. This is done by setting
+a `Genesis Seed` value in your values.yaml file. This will cause Sawtooth to
+delete all existing data and keys on startup and rerun genesis.
+
+The Sawtooth helm chart, which is a dependency of the
+chronicle-on-sawtooth chart, sets a default value for `sawtooth.genesis.seed`
+
+You can set this in your values.yaml file as follows, using a
```
Comment (mtbc) [inline docs/resetting.md:38]:
```suggestion
You can set this in your `values.yaml` file as follows, using a
```

File: docs/restarting.md (hunk 2 above)
Comment (alexhq) [reply docs/restarting.md]: Thanks, no it's a duplicate and I've removed it

File: docs/resetting.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+Next you need to delete the Sawtooth data. This is done by changing the
+value of `Genesis Seed` in your `values.yaml` file.
+This will cause Sawtooth to delete all existing data and keys on startup,
+then rerun genesis.
+
+The Sawtooth helm chart, which is a dependency of the
+chronicle-on-sawtooth chart, sets a default value for `sawtooth.genesis.seed`
+
+If you set a new random string for the genesis seed in your values.yaml file
```
Comment (mtbc) [inline docs/resetting.md:39]:
```suggestion
If you set a new random string for the genesis seed in your `values.yaml` file
```

File: docs/resetting.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
+# Resetting Chronicle
+
+## How to reset a Chronicle Helm deployment
+
+Warning: This will delete all data in Chronicle deployment.
```
Comment (mtbc) [inline docs/resetting.md:5]:
```suggestion
Warning: This will delete all data in the Chronicle deployment.
```

File: docs/resetting.md (hunk 4 above)
Comment (alexhq) [reply docs/resetting.md]: Corrected, thanks

File: docs/resetting.md (hunk 5 above)
Comment (alexhq) [reply docs/resetting.md]: Also corrected

**Review by mtbc — APPROVED** [review]
